src/Build/Evaluation/Expander.cs

```diff
@@ -3142,15 +3143,15 @@ internal string Receiver
                 // A static method is the content that follows the last "::", the rest being the type
                 if (propertyValue == null && expressionRoot[0] == '[')
                 {
-                    var typeEndIndex = expressionRoot.IndexOf(']', 1);
+                    var typeEndIndex = expressionRoot.IndexOf(']');
```

Comment:
Missing overload on Span?

Reply:
We have this extension method in [MonoDevelop](http://source.monodevelop.com/#MonoDevelop.Core/MemoryExtensions.cs,949db08fee756f03)

Reply:
The original code I wrote here was `var typeEndIndex = expressionRoot.Slice(1).IndexOf(']');` (which is also what the MonoDevelop source code would do).

I opted for the non-sliced version assuming that letting IndexOf iterate over index 0 is more efficient than performing a slice with a startIndex of 1 each time.